flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -131,6 +131,9 @@
 		"Remote connection to [null] failed with java.nio.channels.NotYetConnectedException",
 		"java.io.IOException: Connection reset by peer",
 
+		// filter out expected ResourceManagerException caused by intended shutdown request
+		"Received shutdown request from YARN ResourceManager.",
```

Comment:
```suggestion
		YarnResourceManager.ERROR_MASSAGE_ON_SHUTDOWN_REQUEST,
```

Reply:
Thanks! I just forgot that there is already a static constant overwhelming literal :P